src/app/types/request.type.ts

```diff
@@ -0,0 +1,7 @@
+import { Request } from 'express';
```

Comment:
I think these types should create a separate file since cannot be shared. 

You will have multiple request handlers for multiples verbs, you cannot share the query params from a `get` in the `post`.